starter-code/index.html

```diff
@@ -4,39 +4,68 @@
   <meta charset="UTF-8">
   <title>Spotify</title>
 </head>
+<link rel="stylesheet" href="./css/main.css">
 <body>
-Premium Discover Help Download
+<header>
+  <navbar>
+      <img src = "./images/spotify-logo.png">
+      <ul> 
+          <li><a href = '#'> Premium</a></li>
+          <li><a href = '#'> Discover</a></li>
+          <li><a href = '#'> Help</a></li> 
+          <li><a href = '#'> Download</a></li>
+      </ul>
+  </navbar>
+</header>
 
-Music for everyone.
-Spotify is now free on mobile, tablet and computer.
-Listen to the right music, wherever you are.
+<!-- <hr> -->
+<div class = "large-image-with-background-text">
```

Comment:
You are not writing any semantic tags in your HTML code? 
You are also forgetting to include `<main>` and  `<section>` tags. The `<section>` element groups related content together, typically each section would have its own heading.